department-of-veterans-affairs/va.gov-cms
# VACMS-4628: Add revision log message validation to vcs.

## Description
See #4628

## Testing done
Visual

## QA steps
As an administrator, edit any existing vc node type without adding a revision log message. 
Visually verify that a form error is thrown, indication you need to add a revision log message.
Add a message, and confirm that form saves without error being thrown.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
